docs/demo-data/organization-activity.tsv

```diff
@@ -0,0 +1,7 @@
+date	last four weeks	last week	last day
+2018-01-23	26	23	18
+2018-01-22	26	22	16
+2018-01-21	26	21	13
+2018-01-20	26	22	16
+2018-01-19	26	23	17
+2018-01-18	26	23	12
```

Comment:
Maybe, we could add some more data points?

Reply:
I would argue that this is enough to see how the chart would look like. The chart is not that unusual to warrant more data I would say 😄

Reply:
Okay, that was just a suggestion anyways 🙂.